### PR TITLE
Added location based on IP Address Script

### PR DESCRIPTION
**Pull request by pankaj892**
#666 resolved

### CODE REVIEW

**Comment by pankaj892** [comment]
@HarshCasper review the PR and merge the changes

**Comment by pankaj892** [comment]
I see the PR hasn't been merged yet is there any issue with the PR @HarshCasper

File: README.md [hunk 1]
```diff
@@ -118,6 +118,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
...
     <td align="center"><a href="https://www.linkedin.com/in/sharvari-raut-a62a99171"><img src="https://avatars1.githubusercontent.com/u/57194200?v=4" width="100px;" alt=""/><br /><sub><b>Sharvari Raut</b></sub></a><br /><a href="https://github.com/HarshCasper/Rotten-Scripts/commits?author=sharur7" title="Code">💻</a> <a href="https://github.com/HarshCasper/Rotten-Scripts/commits?author=sharur7" title="Documentation">📖</a></td>
+    <td align="center"><a href="https://github.com/pankaj892"><img src="https://avatars3.githubusercontent.com/u/31444506?v=4" width="100px;" alt=""/><br/><sub><b>Pankaj Biradar</b></sub></a><br /><a href="https://github.com/HarshCasper/Rotten-Scripts/commits?author=pankaj892" title="Documentation">📖 💻 👨‍💻</a></td>
```
Comment (HarshCasper) [inline README.md:121]: You don't need to make this change manually. The All Contributors Bot does this for us, automatically.

File: Python/Get_location_based_on_IP_Address/locationip.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+from ipregistry import IpregistryClient
+
+client = IpregistryClient("tryout")  
+ipInfo = client.lookup() 
+print(ipInfo)
```
Comment (HarshCasper) [inline Python/Get_location_based_on_IP_Address/locationip.py:5]: Add a new line at the EOF.

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Do we actually need to have a script for this? 

It's just a moderate import and use of pre-built function to try out the lookup.

**Comment by pankaj892** [comment]
@HarshCasper  i have made the necessary changes please review them

**Comment by HarshCasper** [comment]
I am still not able to understand the purpose of the Script. Also, you have not added any README to show how the Script works.

**Comment by pankaj892** [comment]
@HarshCasper  the Script gives the location of a user along with details like Country Code,Telecom Operator based on the user's IP Address

**Comment by pankaj892** [comment]
@HarshCasper I see the PR hasn't been merged did i do any mistake

**Comment by HarshCasper** [comment]
> @HarshCasper I see the PR hasn't been merged did i do any mistake

Make README changes and add an apt description about your Script with some screenshots as well

**Pull request closed without merging**